### ISSUE TITLE
Global settings for all projects

### ISSUE THREAD

**Issue by pedromartinez11** [ben-gibson/GitLink#116]
**Is your feature request related to a problem? Please describe.**
I have to make the same changes for all projects, it would be nice to be able to specify defaults for all projects.
Personally, I always use these settings for all my projects:
- Check commit exists on remote: uncheck
- Host: GitLab

**Describe the solution you'd like**
Add global GitLink that is used as the default for all projects.

**Describe alternatives you've considered**
I manually set the settings for each project (too many!)

**Additional context**

**Comment by kubk** [comment]
Completely agree. Each time I open a new repo I have to switch from GitHub to GitLab. It would be nice to select it only once globally.

**Comment by ben-gibson** [comment]
Yeah I think generally the plugin settings need to be re-worked. Ideally, it would look at all the `VCS` roots in the project and allow the plugin to be configured for each one independently. Then it would have global, default settings, that would be used for any of the options not explicitly set for a `VCS` root. This would be a fair amount of work though.

**Comment by ben-gibson** [comment]
Captured in the new issue #121
